### PR TITLE
MAJ-1156_update copyright issues for all documentation module in all Geomajas projects

### PR DESCRIPTION
**Pull request by streulma**

### CODE REVIEW

File: plugin/layer-common/layer-common-documentation/src/docbkx/master.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 	<bookinfo>
 		<copyright>
-			<year>2014</year>
+			<year>2015</year>
 			<holder>Geosparc nv</holder>
```
Comment (janvenstermans) [inline plugin/layer-common/layer-common-documentation/src/docbkx/master.xml]: should be: <year>2014-2015<year>

File: plugin/layer-common/layer-common-documentation/src/docbkx/master.xml [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 	<bookinfo>
 		<copyright>
-			<year>2014</year>
+			<year>2013-2015</year>
 			<holder>Geosparc nv</holder>
```
Comment (janvenstermans) [inline plugin/layer-common/layer-common-documentation/src/docbkx/master.xml]: this is correct

**Comment by janvenstermans** [comment]
Jan Venstermans: +1

**Comment by jandm** [comment]
Jan De Moerloose: +1 (binding)

**Comment by oliverm-be** [comment]
Oliver May: +1 (binding)

**Comment by janvenstermans** [comment]
Vote successful:
Jan De Moerloose: +1 (binding)
Oliver May: +1 (binding)

Jan Venstermans: +1

**Pull request merged**